src/Test/contlinear.jl

```diff
@@ -1153,6 +1184,13 @@ function linear10test(model::MOI.ModelLike, config::TestConfig)
             @test MOI.get(model, MOI.DualStatus()) == MOI.FEASIBLE_POINT
             @test MOI.get(model, MOI.ConstraintDual(), c) ≈ 1 atol=atol rtol=rtol
         end
+
+        if config.basis
+            # multiple optimal basis, x or y can in the optimal basis
```

Comment:
Nit: "There are multiple optimal bases. Either x or y can be in the optimal basis." (http://www.juliaopt.org/JuMP.jl/dev/style/#Comments-1)
Applies below also.